ThemeSetup.php

```diff
@@ -139,52 +147,6 @@ public static function init_gutenberg_config() {
 
 		// Default Gutenberg styles on Frontend.
 		add_theme_support( 'wp-block-styles' );
-
-		// Disable Font Sizes by default.
-		add_theme_support( 'editor-font-sizes' );
-		add_theme_support( 'disable-custom-font-sizes' );
-
-		// Disable Colors by default.
-		add_theme_support( 'editor-color-palette' );
```

Comment:
This is also required  in case the developer doesn't want to display the color palette panel.

Reply:
this was moved to in backend/WP/Gutenberg/Editor.php